packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -247,6 +274,8 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
   static const Duration _defaultHoverShowDuration = Duration(milliseconds: 100);
   static const Duration _defaultWaitDuration = Duration.zero;
   static const bool _defaultExcludeFromSemantics = false;
+  static const TooltipTriggerMode _defaultTriggerMode = _kTooltipTriggerMode;
```

Comment:
It doesn't look like we actually need _kTooltipTriggerMode, you can just set it here. It does not appear to be used anywhere else.

Reply:
I thought I mentioned that before, but the reason for the constant isn't to avoid duplicate uses, but to give flutter's users the ability to reference the default value by reference, for any reason (much like `kToolbarHeight`, although that one is a number)

Looking at it though I now see I made it a private constant, where I should of made it public.
Assuming that's fine by you?

Reply:
What purpose does it serve to have `kTooltipTriggerMode`? Users can just specify `TooltipTriggerMode.longPress`.

Reply:
What if we decide to change the default at some point in the future? 
From the user perspective, specifying `TooltipTriggerMode.longPress` would be specifying the default value.
specifying `kTooltipTriggerMode` would be specifying the default behavior, whatever it may be.

Reply:
That may actually make it harder to change, if we were to do so, in the future. If users rely on the value of `kTooltipTriggerMode`, they would be broken if we changed it. I don't feel that it is necessary. It is only being used to set the default of the widget, which is where we would potentially need to change something in the future anyways.

Reply:
The idea  of using `kTooltipTriggerMode` is when you want to rely on the framework default behavior, keeping in mind that if it changes - you change with it, without the need to actually set it.

But sure, if you don't like it I won't do it :)